src/main/java/com/prajina/academy/controller/CourseRestController.java

```diff
@@ -79,8 +79,10 @@
 			
 			return new ResponseEntity<Course>(HttpStatus.NO_CONTENT);
```

Comment:
good HttpStatus usage (204)  here. Keep Response Entity statuses in the controller to 2xx and 5xx classes.